extensions/che-theia-hosted-plugin-manager-extension/package.json

```diff
@@ -12,7 +12,7 @@
     "@theia/core": "next",
     "@theia/plugin-dev": "next",
     "@theia/plugin-ext": "next",
-    "@eclipse-che/workspace-client": "latest",
+    "@vzhukovs/workspace-client": "0.0.9",
```

Comment:
Should it be replaced on `@eclipse-che/workspace-client` before merging?

Reply:
Yes, it will be replaced before merge